include/xtensor/xtags.hpp

```diff
@@ -0,0 +1,69 @@
+/***************************************************************************
+* Copyright (c) 2017, Ullrich Koethe                                       *
+*                                                                          *
+* Distributed under the terms of the BSD 3-Clause License.                 *
+*                                                                          *
+* The full license is in the file LICENSE, distributed with this software. *
+****************************************************************************/
+
+#ifndef XTAGS_HPP
+#define XTAGS_HPP
+
+/**********************************************************/
+/*                                                        */
+/*     helper for keyword arguments and related stuff     */
+/*                                                        */
+/**********************************************************/
+
+namespace xt
+{
+
+    /// Determine size of an array type at runtime.
+static const int runtime_size  = -1;
+
+    /// Don't initialize memory that gets overwritten anyway.
+enum skip_initialization_tag { dont_init };
+
+    /// Copy-construct array in reversed order.
```

Comment:
I know it's nitpicky, but it would be cool to have a similar style everywhere. We don't use these indented triple-slash comments. :)